include/glow/Graph/Graph.h

```diff
@@ -221,7 +221,7 @@ class Module final {
   uint64_t getConstantsSize();
 
   /// \Returns the module log context.
-  ModuleLogContext &getModuleLogContext() { return moduleLogCtx_; };
+  LogContext *getModuleLogContext() { return &moduleLogCtx_; };
```

Comment:
Why is this a pointer instead of a reference now? It always points to moduleLogCtx_, right?

Reply:
The LogContext as owned by a Function is a pointer, and so it makes sense for interfaces that handle it to be a pointer as well, we could take a reference here but we'd have to convert to a ptr later anyway.